resources/asciidoctor/lib/docbook_compat/convert_admonition.rb

```diff
@@ -56,16 +56,32 @@ def skip_inline_admonition(node)
     end
 
     def convert_inline_admonition_for_real(node)
-      title_classes =
-        "Admonishment-#{node.attr 'title_type'} #{node.attr 'title_class'}"
-      [
-        %(<span class="Admonishment Admonishment--#{node.type}">),
-        %([<span class="#{title_classes}">#{node.attr 'title'}</span>]),
-        '<span class="Admonishment-detail">',
-        node.text,
-        '</span>',
-        '</span>',
-      ].join "\n"
+
+      # If creating a defintiion admonition, maintain normal style
+      if node.type == 'definition'
+        # title_classes =
+        #   "Definition-#{node.attr 'title_type'} #{node.attr 'title_class'}"
+        [
+          %(<span class="Definition Definition--#{node.type}">),
+          %(<span class="Definition-word">#{node.attr 'title'}</span>),
+          '<span class="Definition-defined">',
+          node.text,
+          '</span>',
+          '</span>',
+        ].join "\n"
+      # For all other admonitions, add custom admonishment style
```

Comment:
I'm not sure if adding an `if` statement here is the right way to go about this. Would it be better to have a new `dockbook_compat` file named `convert_definition`?

Reply:
Removed `if` statement